apps/client-ui/scripts/health-check-preview.sh

```
@@ -0,0 +1,314 @@
+    
+    # Check for error indicators
+    if echo "$content" | grep -qi "error\|exception\|failed"; then
+        print_warning "Potential error content detected in page"
+    fi
+    
+    return 0
+}
+
+# Function to check API connectivity
+check_api_connectivity() {
+    local preview_url="$1"
```

Shellcheck (0.10.0)

[warning] preview_url appears unused. Verify use (or export if used externally).

(SC2034)